config_defaults.yml

```diff
@@ -556,46 +556,66 @@ Rakefile:
 Gemfile:
   required:
     ':development':
-      # hardcode JSON version to what's shipped in pdk for now
-      - gem: json
-        version: '= 2.0.4'
-        condition: "Gem::Requirement.create('~> 2.4.2').satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.1.0'
-        condition: "Gem::Requirement.create(['>= 2.5.0', '< 2.7.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.3.0'
-        condition: "Gem::Requirement.create(['>= 2.7.0', '< 2.8.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: 'puppet-module-posix-default-r#{minor_version}'
+      - gem: 'json'
+        version: '~> 2.0'
+      - gem: 'voxpupuli-puppet-lint-plugins'
+        version: '~> 3.0'
+      - gem: 'facterdb'
+        version:
+        - '>= 0.8.1'
+        - '< 2.0.0'
+      - gem: 'metadata-json-lint'
+        version:
+        - '>= 2.0.2'
+        - '< 4.0.0'
+      - gem: 'puppetlabs_spec_helper'
+        version:
+        - '>= 2.9.0'
+        - '< 4.0.0'
+      - gem: 'rspec-puppet'
+        version:
+        - '>= 2.3.2'
+        - '< 3.0.0'
+      - gem: 'rspec-puppet-facts'
+        version:
+        - '>= 1.10.0'
+        - '< 3.0.0'
+      - gem: 'codecov'
+        version: '~> 0.2'
+      - gem: 'dependency_checker'
+        version: '~> 0.2'
+      - gem: 'parallel_tests'
+        version: '~> 3.4'
+      - gem: 'pry'
+        version: '~> 0.10'
+      - gem: 'simplecov-console'
+        version: '~> 0.5'
+      - gem: 'puppet-debugger'
         version: '~> 1.0'
+      - gem: 'rubocop'
+        version: '= 1.6.1'
+      - gem: 'rubocop-performance'
+        version: '= 1.9.1'
+      - gem: 'rubocop-rspec'
+        version: '= 2.0.1'
+      - gem: 'ed25519'
+        version: '~> 1.2'
         platforms: ruby
-      - gem: 'puppet-module-posix-dev-r#{minor_version}'
+      - gem: 'bcrypt_pbkdf'
         version: '~> 1.0'
         platforms: ruby
```

Comment:
I'd suggest to drop both ed25519 and bcrypt_pbkdf since net-ssh already depends on them and you don't need them anywhere else.

Reply:
already done